.clang-format

```diff
@@ -0,0 +1,74 @@
+Language:        Cpp
```

Comment:
NTFS has symbolic links so I'd expect it to work on Windows too, but not 100% sure.